finance.py

```diff
@@ -552,9 +552,10 @@ def update_toolbar(self):
             text_next_period = _('Next Year')
 
         # Update the label self.period to reflect the period.
```

Comment:
The comment was for `get_this_period`. Please move your changes above the same

Reply:
@pro-panda thanks. I fixed it!